pkg/api/v1/types.go

```diff
@@ -114,8 +117,10 @@ type VMList struct {
 	Items           []VM            `json:"items"`
 }
 
-// VMSpec is a description of a VM
+// VMSpec is a description of a VM. Not to be confused with api.DomainSpec in virt-handler.
+// It is expected that this structure will be merged with api.DomainSpec as they contain the same fields.
```

Comment:
Oh no, it will be merged with v1.VMSpec (all fields go there). api.DomainSpec and v1.VMSpec will be significantly different in many parts in the future. Basically all host related fields will be removed from v1.VMSpec and instead it will gain much more capabilities for referencing k8s cluster level resources.